keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,389 @@
+non-goal corresponds partially to the [Changing enhancements
+graduation](#changing-enhancements-graduation) section.
+
+#### Accelerated release cycles
+
+The intent of this proposal is to create more opportunities to provide a
+high-value experience for Kubernetes consumers.
+
+The Kubernetes community faces a reasonable amount of tech debt across
+infrastructure, testing, policy, and documentation. This KEP proposes that we
+spend more time paying down that debt.
+
+SIG Release currently produces releases at the following cadence:
+
+- patch releases (`x.y.Z`): [monthly](https://git.k8s.io/sig-release/releases/patch-releases.md)
+- minor releases (`x.Y.z`): [every four months](https://git.k8s.io/sig-release/release-engineering/versioning.md)
+- pre-releases (`x.y.0-(alpha|beta|rc).N`): every 1-3 weeks during active
+  development cycles ([example](https://git.k8s.io/sig-release/releases/release-1.21/README.md#timeline))
+
+At the time of writing, SIG Release considers these to be reasonable cadences
+for patch and pre-releases.
+
+If you'd like to provide suggestions on longer-term improvements that could
+potentially accelerate production of releases, please join the discussion
+[here](https://github.com/kubernetes/sig-release/discussions/1495).
+
+#### Establishing maintenance/stability releases
+
+Establishing a shorter maintenance/stability release at the end of the year has
+been casually discussed at several points in the project, with the most recent
+(at the time of writing) occurrence being
+[here](https://github.com/kubernetes/sig-release/issues/809).
+
+Nothing compelling has emerged from previous conversations to give cause to
+establish maintenance/stability releases.
+
+Fixing bugs, stabilizing components, adding/deflaking tests, improving
+documentation, and graduating features are activities that can and should
+happen in a reasonably consistent manner throughout the year.
+
+#### Determining an upper bound for Release Team shadows
+
+It was noted that fewer releases for the year would lead to fewer opportunities
+to participate on the Release Team.
+
+This will be discussed and addressed in
+https://github.com/kubernetes/sig-release/issues/1494.
+
+## Proposal
+
+The following tables detail a notional timeline for the remainder of 2021 and 
+for 2022, leveraging the historical *4-releases-per-year cadence*. Generally, 
+code freeze remains in effect until the last week of the release, so 
+development for the next release generally starts prior to the official release 
+team kickoff. A minimum of 1 week is needed between releases to fully form the 
+release team and to facilitate on-boarding of shadows. The fourth release of 
+the year has traditionally been compressed and limited in scope, overlapping 
+with end of year holidays and vacation for many contributors. Additionally, 
+KubeCon normally occurs during at least one release, eliminating a week of 
```

Comment:
I think that counting Kubecons as only one work of working time spent is a bit low, because of the extent to which very active contribtors to the project are probably very active Kubecon participants.

There's the week before of "get everything ready for the big show", and the week after of "integrate everything you learned while you were there into your todo list", where people probably aren't as productive as they normally would be.

That's really important for the timing calculations below.

Reply:
Indeed, the policy states:

```
Events like KubeCon will be considered as blocked from development or decision-making from the SIG release perspective. SIG Release will also consider the week before and after the event in the same way.
```

I'll add language to state that the release team will not meet during that blocked week and clarify the other two weeks. In the past we didn't break, and I think this change has been beneficial but it consumes time. 

Thanks @youngnick I'll fold this in.